tests/fixtures/detections/example_set_duplicates.yml

```diff
@@ -17,8 +17,8 @@ SummaryAttributes:
   - example_dupe
   - example_dupe
 OutputIds:
-  - 1234
-  - 1234
+  - "1234"
+  - "1234"
```

Comment:
This was actually causing this to be invalid because of a type check rather than duplicate set check.